defaults/validate_test.go

```diff
@@ -18,6 +19,8 @@ func TestValidate(t *testing.T) {
 		MaxUsers      int    `default:"1000" validate:"range:(0,]"`
 		LogLevel      string `default:"ERROR" validate:"oneof:{TRACE, INFO, WARN, ERROR}"`
 		S3URI         string `default:"" validate:"s3uri"`
+		LicenseFile   string `default:"" validate:"empty;file"`
```

Comment:
I recommend `|` instead of `;`. Then we could add `&` for improved boolean logic :)

Reply:
Oooh, I like it! Let's do that :)

Reply:
that was half a joke you know :p

Reply:
You have to be careful with your jokes!